assets/src/data/model/datetime/index.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * External imports
+ */
+import moment from 'moment';
+import { isUndefined } from 'lodash';
+import PropTypes from 'prop-types';
+
+import {
+	getQueryString as baseGetQueryString,
+} from '../base';
+
+export const nowDateAndTime = moment();
+
+/**
+ * Described attributes for this model
+ * @type {{attributes: *}}
+ */
+export const queryDataTypes = {
+	queryData: PropTypes.shape( {
+		limit: PropTypes.number,
+		orderBy: PropTypes.oneOf( [
+			'DTT_name',
+			'DTT_ID',
+			'start_date',
+			'end_date',
+		] ),
+		order: PropTypes.oneOf( [ 'asc', 'desc' ] ),
```

Comment:
should we consider the values `'ASC'` and `'DESC'` as well 

and/or do one of the following:

- use a function to convert all incoming values for `order` to lowercase (or uppercase)
- use constants like `QUERY_ORDER_ASC` and `QUERY_ORDER_DESC` instead

Reply:
ya will do.